bindings/generate_bindings_py.py

```diff
@@ -426,6 +427,17 @@ def __init__(self, operation_name: str, error: "runtimeStreamError") -> None:
     def __str__(self) -> str:
         return self.message
 
+
+class DetEnum(enum.Enum):
+    def __str__(self) -> str:
+        skip = len(self.prefix())
+        return f"{self.value[skip:]}"
```

Comment:
so, the output of `str(enumval)` could not be fed back into `EnumClass(str(enumval))`?

That seems confusing

Reply:
hmm is that a use case we use 🤔 ? I was going off of https://docs.python.org/3/reference/datamodel.html#object.__str__

Reply:
maybe just write a helper function that takes an enum value and returns a string with the prefix stripped?  So the enum stays and enum and has no special properties, but enum_value_unprefixed(state) returns ACTIVE instead of STATE_ACTIVE?

or maybe have DetEnum offer a unprefixed_value() method, rather than overwriting an existing method?

Reply:
we had a chat offline and figured out we can't do `EnumClass(str(enumval))` in `main` today either so won't lose any functionality there. Also since overriding `__str__`  for this use case seems Pythonic (in line with the docs) we're going with overriding `__str__` for our generated enums.